src/routes/stake/SwapInput.svelte

```diff
@@ -1,6 +1,6 @@
 <script lang="ts">
-	import { Asset, handleInput } from '$lib';
-	import { inputValue, user } from '$lib/stores';
+	import { Asset } from '$lib';
+	import { inputAmount, user, handleInputAmount } from '$lib/stores';
```

Comment:
There is one input amount for the whole app?

Reply:
Initially, I built this store for the wallet because I needed it in several components. But I didn't want to write the same code for another variable and because you cannot use the input value in the SNS and in the wallet at the same time, I reused the store.

Reply:
Initially, I built this store for the wallet because I needed it in several components. But I didn't want to write the same code for another variable and because you cannot use the input value in the SNS and in the wallet at the same time, I reused the store.